egs2/TEMPLATE/asr2/README.md

```diff
@@ -0,0 +1,187 @@
+# ESPnet2 ASR2 Recipe TEMPLATE
+
+This is a template of ASR2 recipe for ESPnet2.
+The difference from ASR1 is that discrete tokens are used as input instead of conventional audios / spectrum features.
+
+## Table of Contents
+
+* [ESPnet2 ASR2 Recipe TEMPLATE](#espnet2-asr2-recipe-template)
+  * [Table of Contents](#table-of-contents)
+  * [Recipe flow](#recipe-flow)
+    * [1\. Data preparation](#1-data-preparation)
+    * [2\. Speed perturbation](#2-speed-perturbation)
+    * [3\. Wav format](#3-wav-format)
+    * [4\. Generate discrete tokens](#4-generate-discrete-tokens)
+    * [5\. Generate dump folder](#5-generate-dump-folder)
+    * [6\. Removal of long / short data](#6-removal-of-long--short-data)
+    * [7\. Input / Output Token list generation](#7-input-output-token-list-generation)
+    * [8\. LM statistics collection](#8-lm-statistics-collection)
+    * [9\. LM training](#9-lm-training)
+    * [10\. LM perplexity](#10-lm-perplexity)
+    * [11\. Ngram-LM training](#11-ngram-lm-training)
+    * [12\. ASR statistics collection](#12-asr-statistics-collection)
+    * [13\. ASR training](#13-asr-training)
+    * [14\. ASR inference](#14-asr-inference)
+    * [15\. ASR scoring](#15-asr-scoring)
+    * [16\-18\. (Optional) Pack results for upload](#16-18-optional-pack-results-for-upload)
+  * [How to run](#how-to-run)
+    * [LibriSpeech training](#librispeech-training)
+  * [Related works](#related-works)
+
+## Recipe flow
+
+ASR2 recipe consists of 15 stages.
+
+### 1. Data preparation
+
+Data preparation stage.
```

Comment:
I think more information can be provided about this stage, because the used data files are different from the standard ones in Kaldi.

Reply:
Can you be more specific? How different from standard ones is it?

Reply:
For example, no `wav.scp` is used/generated. Some new files in addition to the standard files (wav.scp, text, utt2spk, spk2utt) are generated.

Reply:
In recent commits, stage 1 generates wav.scp and used in stage 2 - 4. New files, to which I supposed you referred are the discrete token input text and the renamed target text, are generated in stage 4b now.